pkg/service/sniffer/runtime/crio.go

```diff
@@ -0,0 +1,41 @@
+package runtime
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type CrioBridge struct {
+}
+
+func NewCrioBridge() CrioBridge {
+	return CrioBridge{}
+}
+
+func (c CrioBridge) NeedsPid() bool {
+	return true
+}
+
+func (c CrioBridge) BuildInspectCommand(containerId string) []string {
+	return []string{"chroot", "/host", "crictl", "inspect",
+		"--output", "json", containerId}
+}
+
+func (c CrioBridge) ExtractPid(inspection string) (*string, error) {
+	var result map[string]interface{}
+	json.Unmarshal([]byte(inspection), &result)
+	pid := fmt.Sprintf("%.0f", result["pid"].(float64))
+	return &pid, nil
+}
+
+func (c CrioBridge) BuildTcpdumpCommand(containerId *string, netInterface string, filter string, pid *string) []string {
+	return []string{"nsenter", "-n", "-t", *pid, "--", "tcpdump", "-i", netInterface, "-U", "-w", "-", filter}
+}
+
+func (c CrioBridge) BuildCleanupCommand() []string {
+	return nil // No cleanup needed
+}
+
+func (c CrioBridge) GetDefaultImage() string {
+	return "registry.access.redhat.com/rhel7/support-tools"
```

Comment:
The `corfr/tcpdump` image could be used but the `nsenter` seems very broken. I used this Red Hat image since I know it works fine and is regularly updated. However, it is quite large. 

Are there any other images that include tcpdump and nsenter we can use?

Reply:
I just built a new image based on corfr's here: `docker.io/maintained/tcpdump`. It's updated nsenter is way less buggy if you want to use it.

Reply:
Great, can we re-use it for docker runtimes as well? I prefer to have a single image for all runtimes if possible

Reply:
Resolved